apps/ui/src/composables/useAccount.ts

```diff
@@ -16,15 +16,14 @@ watch(
 
 export function useAccount() {
   async function loadVotes(networkId: NetworkID, spaceIds: string[]) {
-    votes.value = {};
-
     const account = web3.value.account;
-    if (!account) return;
+    if (!account) {
+      votes.value = {};
+      return;
+    }
```

Comment:
Above we have `watch` that looks at `web3.value.account` changing and reacting to it by clearing `pendingVotes`. Do you think we can just move this clearing to that watch (maybe then we don't have to only clear if for guest users, but always when it changes.

This way we have single place for clearing both values.